src/services/nats.ts

```diff
@@ -34,8 +38,45 @@ export interface AirlockPayload {
     query: JSONValue;
 }
 
+interface AirlockJWT {
+    uid: string;
+    cid: string;
+    sub?: string;
+    studio?: boolean;
+}
+
 let natsConnection: NatsConnection;
 
+export function parseJwtToNats(jwt: string): MsgHdrs {
+    if (!jwt) {
+        throw new Error("INVALID_JWT");
+    }
+
+    const decoded = jwtDecode<AirlockJWT>(jwt.split(" ")[1]);
+    const natsHeaders = headers();
+
+    natsHeaders.set("studio_id", decoded?.cid || "");
+    natsHeaders.set("user_id", decoded?.uid || "");
+    natsHeaders.set("username", decoded?.sub || "");
+    natsHeaders.set("is_studio", (decoded?.studio || "").toString());
```

Comment:
we should use the `String` constructor to cast a type to a String instead of the the `toString()` method. always use the constructor function for a given type for type casting, they've been standardized to have a proper, consistent way of doing type casting 👍

Reply:
Oh! I didn't knew that, thank you !